### DISCUSSION TITLE
Missing GetSystemMetrics-Hook.dll when building from repo

### DISCUSSION THREAD

**Discussion by kienluu78** [gucci-on-fleek/lockdown-browser#62] [Q&A]
Hi guys see the error in the attached picture, whenever I run build.ps1 it comes up with this error not sure if the dll is missing from the repo, or if i'm doing something wrong.

![Lockdown Error](https://github.com/gucci-on-fleek/lockdown-browser/assets/163991341/a76bf6ef-9e48-4aa4-a73b-7f6614ae812c)

**Comment by gucci-on-fleek** [comment]
That's really weird. Up until the `cp` error, the output looks correct (it matches the [demo video](https://github.com/gucci-on-fleek/lockdown-browser?tab=readme-ov-file#lockdown-browser-in-windows-sandbox) exactly), so I'm not too sure what's going on. What are the contents of your `build/` directory?

**Reply by kienluu78** [reply]
Yeah I followed the directions to the letter and for some reason its always this error. See attached picture for the build directory.

![image](https://github.com/gucci-on-fleek/lockdown-browser/assets/163991341/746d16fd-ccfd-4a6b-8c65-c1472e15533c)

**Reply by gucci-on-fleek** [reply]
It looks like it's building a static library for some reason, instead of a shared/dynamic library. But in your screenshot it shows that it was passed the `/dll` flag, so I have no idea why this would be happening. 

Maybe try rerunning the VS Build Tools installer and make sure that you enabled anything that says “shared library support”, “dynamic libraries”, “linker tools” or similar. Then, clone the repo from scratch and try building it again. This is pretty much just a wild guess though.

**Reply by al3naziiy** [reply]
Is it possible to create the build method faster knowing how please explain it to me here in time if necessary
